opentok/opentok-android-sdk-samples
# VIDCS-869: Fix crash when toggling video

#### What is this PR doing?
Fixes an issue where crashing may occur when toggling video

#### How should this be manually tested?
To reproduce, follow the steps detailed in https://github.com/opentok/opentok-android-sdk-samples/issues/461 

To confirm fix, checkout this branch and then follow the same steps as before 🎉
